### PR TITLE
WIP: Adding support page

### PR DESCRIPTION
**Pull request by jim-docker**
Signed-off-by: Jim Ehrismann <jehrismann@mirantis.com>

- [x] define actual content of support page
- [ ] rework it as an extension

image

addresses #742

### CODE REVIEW

File: src/renderer/components/+support/support.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        margin-top: -$margin * 2;
+      }
+    }
+  }
+
+  .Select {
+    &__control {
+      box-shadow: 0 0 0 1px $borderFaintColor;
+    }
+  }
```
Comment (jim-docker) [inline src/renderer/components/+support/support.scss:66]: copy/pasted from preferences, will pare it down when page is finalized

File: src/renderer/components/cluster-manager/bottom-bar.tsx [hunk 2]
```diff
@@ -16,6 +19,13 @@ export class BottomBar extends React.Component {
           <span className="workspace-name">{currentWorkspace.name}</span>
         </div>
         <WorkspaceMenu htmlFor="current-workspace"/>
+        <div id="support" className="flex gaps align-center box right">
+          <Icon 
+            small
+            svg="support"
```
Comment (jim-docker) [inline src/renderer/components/cluster-manager/bottom-bar.tsx:25]: this icon is listed on the material-design-icons website but is not present in the npm repo. There are reports of other icons missing and this repo not being maintained. Have we considered fontawesome instead?

**Comment by jim-docker** [comment]
image

File: src/renderer/components/cluster-manager/bottom-bar.tsx (hunk 2 above)
Comment (ixrock) [reply src/renderer/components/cluster-manager/bottom-bar.tsx:25]: It seems that we should manage latest updates of material-icons-font in our repo or consider to switch to another icon-font library with active maintainers.

**Comment by ixrock** [comment]
Moved to `lensapp:extension_support_page` to continue work as an extension.

**Pull request merged**